spec/helpers/articles_helper_spec.rb

```diff
@@ -53,4 +53,14 @@ def internal_navigation?
       it { is_expected.to include('height="18" width="18"') }
     end
   end
+
+  describe "#formated_date" do
+    it "correctly formats the date when present" do
+      expect(helper.formated_date(Time.now.to_i)).to match(/\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z/)
```

Comment:
I think this needs to be `Time.current` and without `to_i`

Reply:
It works with either, but agreed, `Time.current` is closer to how the AR timestamps look.